test/visual/specs/button.gspec

```diff
@@ -1,200 +1,78 @@
 # http://galenframework.com/docs/reference-galen-spec-language-guide/#Objectdefinition
 
 @objects
-    
-    button_cell_1                               .button-grid__cell:nth-of-type(1) 
-        button_primary_*                        .button:not(.button--icon-only)
-        button_icon                             .button--icon-only
-    button_cell_2                               .button-grid__cell:nth-of-type(2)
-        button_secondary_*                      .button:not(.button--icon-only)
-        button_icon                             .button--icon-only
-
-
-    dark                                            .background--dark
-        button_cell_1                               .button-grid__cell:nth-of-type(1) 
-            button_primary_*                        .button:not(.button--icon-only)
-            button_icon                             .button--icon-only
-        button_cell_2                               .button-grid__cell:nth-of-type(2)
-            button_secondary_*                      .button:not(.button--icon-only)
-            button_icon                             .button--icon-only
-
-
-    light                                           .background--light
-        button_cell_1                               .button-grid__cell:nth-of-type(1) 
-            button_primary_*                        .button:not(.button--icon-only)
-            button_icon                             .button--icon-only
-        button_cell_2                               .button-grid__cell:nth-of-type(2)
-            button_secondary_*                      .button:not(.button--icon-only)
-            button_icon                             .button--icon-only
 
+    button_default_*                                .button:not(.button--icon-only)
+    button_icon_only                                .button--icon-only 
 
 # http://galenframework.com/docs/reference-galen-spec-language-guide/#SpecsReference
 
 = Height =
 
-    @on *
-        button_cell_1.button_primary_*:
-            css height is "56px"
-
-        button_cell_1.button_icon:
-            css height is "56px"
-
-        button_cell_2.button_secondary_*:
-            css height is "56px"
+    @on xs
+        button_default_*:
+            css height is "49px"
 
-        button_cell_2.button_icon:
-            css height is "56px"
+        button_icon_only:
+            css height is "49px"
 
-        dark.button_cell_1.button_primary_*:
+    @on s
```

Comment:
@NathanRodger You could consolidate the tests from s to l with a tag notation of

```
@on s, m, l
    button_default_*:
        css height is "56px"
    button_icon_only:
        css height is "56px"
```